src/server/game/Miscellaneous/Language.h

```diff
@@ -1228,42 +1237,42 @@ enum TrinityStrings
     LANG_OPVP_ZM_GOSSIP_ALLIANCE        = 10054,
     LANG_OPVP_ZM_GOSSIP_HORDE           = 10055,
 
-    LANG_BG_SA_START_TWO_MINUTES            = 10056,
-    LANG_BG_SA_START_ONE_MINUTE             = 10057,
-    LANG_BG_SA_START_HALF_MINUTE            = 10058,
-    // unused                                 10059-10062
-    LANG_BG_SA_ALLIANCE_CAPTURED_RELIC      = 10063, //The Alliance captured the titan portal!
-    LANG_BG_SA_HORDE_CAPTURED_RELIC         = 10064, //The Horde captured the titan portal!
-    LANG_BG_SA_ROUND_TWO_ONE_MINUTE         = 10065, //Round 2 of the Battle for the Strand of the Ancients begins in 1 minute.
-    LANG_BG_SA_ROUND_TWO_START_HALF_MINUTE  = 10066, //Round 2 begins in 30 seconds. Prepare yourselves!
+    LANG_BG_SA_START_TWO_MINUTES        = 10056,
+    LANG_BG_SA_START_ONE_MINUTE         = 10057,
+    LANG_BG_SA_START_HALF_MINUTE        = 10058,
+    //                                  = 10059-10062, not used
+    LANG_BG_SA_ALLIANCE_CAPTURED_RELIC  = 10063, // "The Alliance captured the titan portal!"
+    LANG_BG_SA_HORDE_CAPTURED_RELIC     = 10064, // "The Horde captured the titan portal!"
+    LANG_BG_SA_ROUND_TWO_ONE_MINUTE     = 10065, // "Round 2 of the Battle for the Strand of the Ancients begins in 1 minute."
+    LANG_BG_SA_ROUND_TWO_START_HALF_MINUTE = 10066, // "Round 2 begins in 30 seconds. Prepare yourselves!"
 
     // Use for custom patches             11000-11999
     LANG_AUTO_BROADCAST                 = 11000,
     LANG_INVALID_REALMID                = 11001,
 
     // Show Kick in world
-    LANG_COMMAND_KICKMESSAGE_WORLD = 11002,
+    LANG_COMMAND_KICKMESSAGE_WORLD      = 11002,
 
     // Show Mute in world
-    LANG_COMMAND_MUTEMESSAGE_WORLD = 11003,
+    LANG_COMMAND_MUTEMESSAGE_WORLD      = 11003,
 
     // Show Ban in world
-    LANG_BAN_CHARACTER_YOUBANNEDMESSAGE_WORLD     = 11004,
+    LANG_BAN_CHARACTER_YOUBANNEDMESSAGE_WORLD = 11004,
     LANG_BAN_CHARACTER_YOUPERMBANNEDMESSAGE_WORLD = 11005,
-    LANG_BAN_ACCOUNT_YOUBANNEDMESSAGE_WORLD       = 11006,
-    LANG_BAN_ACCOUNT_YOUPERMBANNEDMESSAGE_WORLD   = 11007,
-
-    LANG_NPCINFO_INHABIT_TYPE                     = 11008,
-    LANG_NPCINFO_FLAGS_EXTRA                      = 11009,
-    LANG_INSTANCE_LOGIN_GAMEMASTER_EXCEPTION      = 11010,
-
-    LANG_CREATURE_NO_INTERIOR_POINT_FOUND         = 11011,
-    LANG_CREATURE_MOVEMENT_NOT_BOUNDED            = 11012,
-    LANG_CREATURE_MOVEMENT_MAYBE_UNBOUNDED        = 11013,
-    LANG_INSTANCE_BIND_MISMATCH                   = 11014,
-    LANG_CREATURE_NOT_AI_ENABLED                  = 11015,
-    LANG_SELECT_PLAYER_OR_PET                     = 11016,
-    LANG_SHUTDOWN_DELAYED                         = 11017,
-    LANG_SHUTDOWN_CANCELLED                       = 11018
+    LANG_BAN_ACCOUNT_YOUBANNEDMESSAGE_WORLD = 11006,
+    LANG_BAN_ACCOUNT_YOUPERMBANNEDMESSAGE_WORLD = 11007,
```

Comment:
Is there any improvement to be gained from removing the enum alignment?
To me, it looks less readable than before, but maybe I have missed something.